### ISSUE TITLE
"MCP_SetSysProdSettings() failed: ffffffff" when trying to fix region.

### ISSUE THREAD

**Issue by AdmiralCurtiss** [GaryOderNichts/recovery_menu#14]
A little backstory here: Some years ago I acquired a WiiU console that someone clearly tampered with in some way: It's an US console, but trying to launch any application on it (including built-in ones like the System Settings or the Internet browser) results in an error message that the game region is incorrect. Since I had no idea how to fix this I just put it into storage at the time.

Fast-forward a few years, https://github.com/GaryOderNichts/udpih released, which gave me some hope of fixing this console. And, indeed, I can boot into this recovery menu using udpih, and there's even an option for the exact issue I'm facing! Selecting the Debug System Region option shows:

```
System region code: USA
Game region code:   --- --- EUR --- --- ---

Installed Wii U Menu: USA
The game region does not match the installed Wii U Menu.

Repair the system by setting the region code to USA?
```

Confirming the repair, however, only results in an error code, and the console behavior remains unchanged:

```
MCP_SetSysProdSettings() failed: ffffffff
```

Any idea what's going on here? Anything else I can try?

**Comment by GaryOderNichts** [comment]
Seems like `MCP_SetSysProdSettings` was behind a debug mode check.
Can you test if it works with [this](https://github.com/GaryOderNichts/recovery_menu/suites/8235343361/artifacts/359582520) build which patches out that check?

**Comment by AdmiralCurtiss** [comment]
That seems to have worked, yes. Thanks!

**Comment by GaryOderNichts** [comment]
Fixed with version 0.4.
